ansible/roles/jenkins-worker/templates/freebsd.initd.j2

```diff
@@ -28,8 +28,8 @@ jenkins_env=" \
   NODE_TEST_DIR=/home/{{ server_user }}/tmp \
   PATH=/usr/local/libexec/ccache:/usr/local/bin:${PATH} \
   JOBS={{ server_jobs|default(ansible_processor_vcpus|default('2')) }} \
-  CC=cc \
-  CXX=c++"
+  CC={{ custom_cc_command|default('cc') }} \
```

Comment:
Not a problem, I initially started down that path and discovered the configuration option.  Figured it was worth the exploration.

My one concern with hard-coding into the template was that it could cause confusion down the road.  Ideally it would be nice to have the compiler command right along-side the package config that installs it.  An issue for another time, but that was the thinking.